packages/inter-protocol/test/smartWallet/test-oracle-integration.js

```diff
@@ -127,7 +186,7 @@ test('admin price', async t => {
     'priceAggregator',
   );
 
-  // The purse has the invitation to get the makers ///////////
+  // The purse has the invitation to get the makers
```

Comment:
Does the reason you removed that apply to line 227 as well?

Reply:
Just cleanliness. Not worth another push.